src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1223,6 +1223,10 @@ public function testIteratorInput()
 
     public function testSimpleInputStream()
     {
+        if (\PHP_VERSION_ID === 70200 && \PHP_EXTRA_VERSION === 'RC6') {
```

Comment:
I doubt this test is correct, https://github.com/php/php-src/commit/5060fc23 was in may, so more versions are affected

Reply:
git diff between rc5 & 6 hinted me towards this...

Reply:
Well, the github UI tells us that the only tag containing it is RC6. The commit was initially done in May, but it does not mean it was merged in may.

Reply:
Indeed, PR from may, merged in November.... (the reason why I never use "merge", but prefer gi am....)